src/raftstore/store/snap.rs

```diff
@@ -165,8 +170,8 @@ pub trait Snapshot: Read + Write + Send {
 // Only used in tests.
 pub fn copy_snapshot(mut from: Box<Snapshot>, mut to: Box<Snapshot>) -> io::Result<()> {
     if !to.exists() {
-        try!(io::copy(&mut from, &mut to));
-        try!(to.save());
+        io::copy(&mut from, &mut to)?;
```

Comment:
Seems all `?` were written by me. But no one told me this before. Because of performance issue?